### ISSUE TITLE
Update from version 3.2.0 to latest 4.1.0 breaks build. Undefined scss variables

### ISSUE THREAD

**Issue by awacode21** [pmowrer/node-sass-json-importer#78]
Hello @pmowrer ,

for my current project we updated the node-sass-json-importer from version 3.2.0 to version 4.1.0 but that causes the build to break with error message of undefined variables. The variables it complains about are normal scss variables within scss files which are not part of any json file (see screenshot). Any idea?
<img width="538" alt="50899264-de6a0380-1411-11e9-8b6d-7844a370d46e" src="https://user-images.githubusercontent.com/30939926/53084183-9c4bdb80-3500-11e9-84f3-46d9146c980d.png">

The project setup is build with grunt. Using node-sass und grunt-sass.
The relevant part in grunt config is the following:

```
const jsonImporter = require('node-sass-json-importer');

module.exports = {
    css : {
        options: {
            importer: jsonImporter,
            style: 'nested',
            precision: 8,
            trace: true,
            lineNumbers: true,
            sourceMap: true,
            sourceComments: true
        },
        files : [{
            expand: true,
            cwd: '<%= package.sourceFolder %>/scss/',
            src: ['**/*.scss'],
            dest: '<%= package.buildFolder %>/css/',
            ext: '.css'
        }]
    }
}

```
It would be unfortunate circumstances if we would be forced to stick with an old version of node-sass-json-importer unable to update.

Best regards, Annick

**Comment by bjoernsteinborn** [comment]
Same for me with 4.1.0

**Comment by acramatte** [comment]
Seems that since 4.0.0 and https://github.com/Updater/node-sass-json-importer/pull/71 you need to call the function on the jsonImporter you import/require

e.g.
```javascript
const jsonImporter = require('node-sass-json-importer');
module.exports = {
    css : {
        options: {
            importer: jsonImporter,
            ...
        },
    }
}
```

should now be

```javascript
const jsonImporter = require('node-sass-json-importer');
module.exports = {
    css : {
        options: {
            importer: jsonImporter(),
            ...
        },
    }
}
```

**Comment by awacode21** [comment]
yes, i saw that in documentation but it has no effect at all. It still breaks with latest version

**Comment by Tylerian** [comment]
IMHO this is an issue with #74 and that commit should be reverted in order to fix it.

The reasons behind my statement is that sass either expects an object literal with a `contents`key in order to load that contents, or an object literal with a `file`key to load that file as though it had been imported directly _([as seen in docs](https://sass-lang.com/documentation/js-api#importer))_. But not an object literal containing both `file` and `contents`.

@pmowrer would you accept a pull request reverting that behavior?

**Comment by pmowrer** [comment]
@Tylerian Can both be supported? E.g. if `file` is present, always use that. Else, use `contents`.

**Comment by pmowrer** [comment]
Closed by #83

**Comment by awacode21** [comment]
Thanks!
